db/migrate/20201217163340_update_enterprise_socia_links.rb

```diff
@@ -1,14 +1,11 @@
 class UpdateEnterpriseSociaLinks < ActiveRecord::Migration
-  def instagram_regex
-    %r{\A(?:https?://)?(?:www\.)?instagram\.com/([a-zA-Z0-9._@-]{1,30})/?\z}
-  end
   def change
```

Comment:
This migration works only in one direction. It's not reversible.
```suggestion
  def up
```